packages/http-server/README.md

```diff
@@ -0,0 +1,63 @@
+# @loopback/http-server
+
+This package implements the HTTP / HTTPS server endpoint for LoopBack 4 apps.
+
+## Overview
+
+This is an internal package used by `RestServer` for creating its HTTP / HTTPS server.
+
+## Installation
+
+To use this package, you'll need to install `@loopback/http-server`.
+
+```sh
+npm i @loopback/htp-server
+```
+
+## Usage
+
+`@loopback/http-server` should be instantiated with an instance of `RestServer`, HTTP / HTTPS options object, and a request handler function.
+
+```js
+import {RestServer} from '@loopback/rest';
+import {Application} from '@loopback/core';
+
+const app = new Application();
+const restServer = new RestServer(app);
+const httpServer = new HttpServer(restServer, {port: 3000, host: ''}, (req, res) => {});
```

Comment:
This example will need to be updated once we remove the dependency from HttpServer to RestServer, see my comments below.

I think it also makes more sense to reorder constructor arguments to accept request handler as the first arg and options as the second.

I would like something along the following lines:

```ts
import {HttpServer} from '@loopback/http-server';

const httpServer = new HttpServer(
  (req, res) => res.end('Hello world'),
  {port: 3000}
);
```